### PR TITLE
Mobile branch

### PR DESCRIPTION
**Pull request by AlphaNtihinduka**
This is the video recorded in Loom describing the project features and the most interesting piece of code👍
https://www.loom.com/share/01553a2ad6ff4df5b5be3c34fe8958b8

Thank you!!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,53 @@
...
+
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ntihinduka-alpha-81bb7b22a/)
+
+## ✍ Contributing 
+Contributions, issues, and feature requests are welcome!
+Feel free to add issues here 👉 https://github.com/AlphaNtihinduka/Capstone-project-1/issues
+
+## Support
+Feel free to give  ✨
+
```
Comment (gemmen29) [inline README.md:47]: - [x] Kindly add an acknowledgment to [Cindy Shin](https://www.behance.net/adagio07) the author of the original design as required by the Creative Commons License in your README file.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    <li class="join"><a href="#"> Join</a> </li>
+    <li class="sponsor"><a href="#">Sponsor</a> </li>
+    <li class="compain"><a href="#"> Tickets</a> </li>
+  </ul>
+</div>
+
+    <section class="banner-area banner-area-about">
+      <div class="greeting_title_banner greeting_title_banner-about">
+      <h2 class="greeting-banner-text">"Hello! Sharing World"</h2>
+      <h1 class="title-banner-text-about">INTERNATIONAL GIRLS <br> INTECH  CONFERENCE <br> 2022</h1>
```
Comment (gemmen29) [inline about.html]: - [ ] Kindly make this header center in the about page in mobile and desktop versions to match the Behance design.

| Your design | Behance design |
| --- | --- | 
| image | image |

File: about.html [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+          <div class="event-card-container">
+            <img src="./images/conference2.jpg" alt="girlsintech">
+            <div class="event-card-description">
+              <h5 class="event-year">November 15, 2022</h5>
+              <p class="event-location">London Hall</p>
+            </div> 
+          </div>
+        </div>
+      </div>
+    </section>
```
Comment (gemmen29) [inline about.html]: ### To match the Behance design:
- [ ] Kindly make `Future Conference Events` center.
- [ ] Kindly make the `text` above the image `white`.
- [ ] Kindly make the `text` in the center of the image in the mobile version.
- [ ] Kindly add the `paragraph` after the heading, you can customize it as you wish. 

| Your design | Behance design |
| --- | --- | 
| image | image |

File: about.html [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+           <img src="./images/airbnb.svg" alt="Airbnb">
+            <img src="./images/amazon-png.png" alt="Amazon">
+            <img src="./images/facebook.svg" alt="facebook">
+          </div>
+          <div class="sponsor-logo">
+            <img src="./images/google.svg" alt="google">
+             <img src="./images/mozilla.svg" alt="mozilla">
+             <img src="https://www.logo.wine/a/logo/Binance/Binance-Vertical-White-Dark-Background-Logo.wine.svg" alt="Binance">
+           </div>
+        </div>
```
Comment (gemmen29) [inline about.html:130]: - [ ] Kindly add the white footer in the mobile version in the about page
image
which is turned to black in the desktop version of the about page
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+           <img src="./images/airbnb.svg" alt="Airbnb">
+            <img src="./images/amazon-png.png" alt="Amazon">
+            <img src="./images/facebook.svg" alt="facebook">
+          </div>
+          <div class="sponsor-logo">
+            <img src="./images/google.svg" alt="google">
+             <img src="./images/mozilla.svg" alt="mozilla">
+             <img src="https://www.logo.wine/a/logo/Binance/Binance-Vertical-White-Dark-Background-Logo.wine.svg" alt="Binance">
+           </div>
+        </div>
```
Comment (gemmen29) [inline index.html:148]: - [ ] Kindly add the white footer that exists on the `index` page in the desktop version.

image

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @AlphaNtihinduka, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @AlphaNtihinduka 👋🏻, 

Good job making the requested changes 💪🏻!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional Changes ♻️ 

- [ ] Kindly position the "Hello! Sharing World" on the home page on the left as seen in the behance design.

| Your Design | Behance Design |
| --- | --- |
| image | image |


- [ ] This section is meant to have only 5 articles/cards that should be placed in a single row as seen in the original behance design.

| Your Design | Behance Design |
| --- | --- |
| image | image |




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**